python/seldon_core/app.py

```diff
@@ -68,11 +69,23 @@ class UserModelApplication(StandaloneApplication):
     user's model.
     """
 
-    def __init__(self, app, user_object, options: Dict = None):
+    def __init__(
+        self, app, user_object, jaeger_extra_tags, interface_name, options: Dict = None
+    ):
         self.user_object = user_object
+        self.jaeger_extra_tags = jaeger_extra_tags
+        self.interface_name = interface_name
         super().__init__(app, options)
 
     def load(self):
+        if self.jaeger_extra_tags is not None:
+            logger.info("Tracing branch is active")
+            from flask_opentracing import FlaskTracing
+
+            tracer = setup_tracing(self.interface_name)
```

Comment:
Interesting. Was this required to account for Gunicorn and multiprocessing?

Reply:
I needed to create the tracer inside each possible path otherwise it didn't work.